varnish-sys/src/vcl/backend.rs

```diff
@@ -351,12 +345,33 @@ unsafe extern "C" fn wrap_pipe<S: Serve<T>, T: Transfer>(
     sc_to_ptr(backend.pipe(&mut ctx, tcp_stream))
 }
 
+// CStr is tied to the lifetime of bep, but we only use it for error messages
+unsafe fn get_type(bep: VCL_BACKEND) -> &'static str {
```

Comment:
why not add this as a method to the `impl VCL_BACKEND` ?

Reply:
that would be smart, wouldn't it?